src/backend/ManagementHub.Storage/ManagementHubDbContext.cs

```diff
@@ -1601,6 +1603,84 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 				.HasConstraintName("fk_tournament_team_participants_team");
 		});
 
+		modelBuilder.Entity<TournamentTeamRosterEntry>(entity =>
+		{
+			entity.ToTable("tournament_team_roster_entries");
+
+			entity.HasIndex(e => new { e.TournamentTeamParticipantId, e.UserId }, "index_roster_entries_on_participant_and_user")
+				.IsUnique();
+
+			entity.Property(e => e.Id).HasColumnName("id");
+
+			entity.Property(e => e.TournamentTeamParticipantId).HasColumnName("tournament_team_participant_id");
+
+			entity.Property(e => e.UserId).HasColumnName("user_id");
+
+			entity.Property(e => e.Role)
+				.HasConversion<int>()
+				.HasColumnName("role");
+
+			entity.Property(e => e.JerseyNumber)
+				.HasMaxLength(5)
+				.HasColumnType("character varying")
+				.HasColumnName("jersey_number");
+
+			entity.Property(e => e.CreatedAt)
+				.HasColumnType("timestamp with time zone")
+				.HasColumnName("created_at");
+
+			entity.Property(e => e.UpdatedAt)
+				.HasColumnType("timestamp with time zone")
+				.HasColumnName("updated_at");
+
+			entity.HasOne(d => d.Participant)
+				.WithMany(p => p.RosterEntries)
+				.HasForeignKey(d => d.TournamentTeamParticipantId)
+				.OnDelete(DeleteBehavior.Cascade)
+				.HasConstraintName("fk_tournament_team_roster_entries_participant");
+
+			entity.HasOne(d => d.User)
+				.WithMany()
```

Comment:
Don't use `WithMany()` without specifying navigation property on the User class, same below